src/main/java/com/manywho/services/dummy/dummy/DummyDatabase.java

```
@@ -15,70 +13,126 @@
 import com.manywho.services.dummy.ApplicationConfiguration;
 
 public class DummyDatabase implements Database<ApplicationConfiguration, Dummy> {
-    
+
     private static HashMap<String, Dummy> DUMMIES;
-    
-    static
-    {
+
+    static {
         DUMMIES = DummyDatabaseDataBuilder.BuildDummies();
     }
 
     @Override
-    public Dummy find(ApplicationConfiguration configuration, ObjectDataType objectDataType, Command command, String id) {   
+    public Dummy find(ApplicationConfiguration configuration, ObjectDataType objectDataType, Command command, String id) {
         return DUMMIES.get(id);
     }
 
     @Override
     public List<Dummy> findAll(ApplicationConfiguration configuration, ObjectDataType objectDataType, Command command, ListFilter filter, List<MObject> objects) {
-        
-        Stream<Dummy> pagedDummies = DUMMIES
-            .values()
-            .stream()
-            .skip(filter.getOffset());
-        
-        if(filter.getLimit() > 0){
-            pagedDummies = pagedDummies.limit(filter.getLimit());
+        return DUMMIES
+                .values()
+                .stream()
+                .filter(c ->
+                {
+                    if (filter == null || filter.getWhere() == null || filter.getWhere().size() == 0) {
+                        return true;
+                    }
+                    String filterColumn = filter.getWhere().get(0).getColumnName();
```

you only support one where element here, could you throw an error if there are more than one in the request, so the user know that is not supported at the moment.

---

I've added error message when filter.getWhere().size() > 1 as suggested.